.github/workflows/ci-cd.yml

```
@@ -41,20 +41,15 @@ jobs:
         os:
         - ubuntu-24.04
         - ubuntu-24.04-arm
-        include:
-        - experimental: false
-        - upload-coverage: false
-        - upload-coverage: true
-          python-version: 3.11
-          os: ubuntu-24.04
       fail-fast: false
     uses: ./.github/workflows/reusable-test.yml
     with:
       python-version: ${{ matrix.python-version }}
       os: ${{ matrix.os }}
-      continue-on-error: ${{ matrix.experimental }}
+      continue-on-error: false
```

`experimental` is currently not used. We might as well remove the `continue-on-error` line.

---

I could see keeping the line if you foresee wanting to re-enable it for certain setups in the future. e.g. if you want to start testing 3.14 betas/rc but not block CI if they break

---

That's exactly what I had been thinking, but we may just add that line again, if that need arises. I'm fine either way.

---

Personally, I implemented inferring this from the Python version string. Like if you have something like `3.14-dev` or `~3.14.0-0`, my things know that these are unstable. And for the explicit flag I call it `xfail` to reference known pytest terms..
I'd keep this in. Perhaps, if you end up trying out my reusable workflow for tox, it has this and I wouldn't be removing the flag, only to re-add it later. It's not as important on the calling side but it could be nice for consistency.

---

This was previously used for a specific combo of OS+python with 3.8+arm so `-dev` would not have been sufficient for that, but it's a fine solution in general. (although we've fucked up something in trio https://github.com/python-trio/trio/pull/3264#discussion_r2088628435)

---

Let's recover this together with my other change.
```suggestion
      continue-on-error: ${{ matrix.experimental }}
```